packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -56,8 +58,29 @@ const l1Networks: { [chainId: number]: L1Network } = {
   }
 }
 
+export type BaseNetwork = L1Network & { isBase: true }
+
+const baseNetworks: { [chainId: number]: BaseNetwork } = {
```

Comment:
`BaseNetwork` and `L1Network` share the same schema, we could possibly rename it to something common.

But I have no idea what 😆 so we may need to go with your option

Reply:
Maybe something like root chain

Reply:
i think technically Base chains are not root chains, terminologies are hard, i wouldn't want to mess it up for now